internal/pkg/pkgfile/build.go

```diff
@@ -44,7 +46,7 @@ func Build(ctx context.Context, c client.Client, options *environment.Options) (
 	}
 
 	loader := solver.BuildkitFrontendLoader{
-		Context: options.GetVariables(),
+		Context: options.ToolchainPlatform.GetVariables(),
```

Comment:
OI wonder if we have to pass platform to buildkit in this function? or it should go with `buildkit` arguments? In the latter case we probably need to add that to the README.

Reply:
Not sure, it seems to be working as is. I can produce the toolchain build for `aarch64`, so maybe neither?

Reply:
yeah, could be, one easy way to tell is to add a new functional test for some simple-as-possible ARM64 build, we just have several ways to call the build process